### ISSUE TITLE
Sound capture mode

### ISSUE THREAD

**Issue by grepwood** [gnif/LookingGlass#99]
It would be great to pass VM sound outputs back to host. Combined with DXGI and #26, we could discontinue the need to use any hardware-based KVMs.

**Comment by Lessaj** [comment]
This is already possible with qemu and not something Looking Glass is intended for.

**Comment by grepwood** [comment]
@Lessaj how so? I've got a VM in libvirt with an ich6 card, but I can't hear anything that's being output by it. I also missed this part in Looking Glass documentation... no, it's not there.

**Comment by Lessaj** [comment]
There has been some work in regards to audio with qemu 3.0 and there are a couple ways to do it, either with spice or directly to PulseAudio but that depends on your system configuration. I'd suggest doing some reason, there is [some information on the Arch Wiki](https://wiki.archlinux.org/index.php/PCI_passthrough_via_OVMF#Passing_VM_audio_to_host_via_PulseAudio) but I found some other topics online as well, slightly dated but still mostly accurate. I'm using libvirt/qemu so I pass the audio directly to PulseAudio. I've used spice in the past as well, it's been a while but it was pretty decent if memory serves.
